.gitignore

```diff
@@ -336,4 +336,6 @@ newVTT.xml
 testground.py
 buildTest.py
 converter.py
-salishan/
\ No newline at end of file
+salishan/
+fixTSI1.py
+sources/vtt_data/vtt_build/from fontTools.ttLib import TTFont.py
```

Comment:
this last line seems weird to me :smile:

Reply:
Shhhhh. I put it in gitignore for a reason! 

I'm somewhat miffed y'all aren't interested in the 'fixTSI1.py' I added too :P

Reply:
I'm definitely interested in that one! I just don't know enough to know what a TSI1 is

(I've since looked and learned some more stuff about VTT!)

Reply:
TSI1 is where VTT puts the assembly stuff compiled from TSI3 (think CVT template, the glyphs' VTT Talk, ...).

Reply:
TSI1 also happens to be where all of the component-related references are stored and mark positioning-related tweaks are made.

Reply:
(I'll put all of my test scripts in a little folder if it'll make y'all feel better. )

Reply:
I don't mind them! I was more worried about the line that looked like some python code that had accidentally made its way in. 😁